src/backofftimer.cpp

```diff
@@ -110,11 +111,16 @@ void BackoffTimer::update(dstime* waituntil)
 {
     if (next)
     {
+        assert(next != 1);
+        if (next == 1)
+        {
+            LOG_warn << "Possible wrong management of timer";
+        }
+
```

Comment:
Once the timeout has expired, in the next SDK loop the corresponding event should be attended and the backoff timer correctly configured. If `next` is still `1` in the next SDK look, that hasn't been correctly made and will probably cause a high CPU usage.